references/documents.md

```diff
@@ -149,21 +147,70 @@ curl \
   "updateId": 1
 }
 ```
-This [update id allows you to track](/references/updates) the current action.
+This `updateId` allows you to [track the current action](/references/updates.md).
 
-## Update documents
+## Add or update documents
 
 <RouteHighlighter method="PUT" route="/indexes/:uid/documents"/>
 
-same as [Add or Update](/references/documents.md#add-or-update-documents)
+Insert a list of documents or update them if they already exist based on [their unique identifiers](/main_concepts/indexes.md#schema-definition).
+
+In case of an update, the old document will be only partially updated according to the fields in the request body. It will not be overwritten entirely.</br>
```

Comment:
```suggestion
In case of an update, the old document will be only partially updated according to the fields in the request body. Only the specified documents fields will be overwritten, missing ones will keep their previous values.</br>
```